docs/guide/quick-start.md

```diff
@@ -61,8 +78,8 @@ class AuthHandler
     {
         $attributes = $this->client->getUserAttributes();
         $email = ArrayHelper::getValue($attributes, 'email');
-        $id = ArrayHelper::getValue($attributes, 'id');
```

Comment:
I'd show a situation where you want to handle both.

Reply:
You mean both id and subject?

Reply:
OpenID and GitHub (id and login).